server.js

```diff
@@ -1,7 +1,25 @@
+require('dotenv').config();
 var express = require('express')
 var app = express()
+const port = 5000;
+const mongoose = require('mongoose');
+
+const uri = 'mongodb+srv://ishita_naraniya:ishhMongodb@cluster1.ponxmip.mongodb.net/?retryWrites=true&w=majority';
+app.get('/', (req, res) => {
```

Comment:
It's a good practice to handle the database connection once when the server starts, not on every request to the root route. This would improve performance and make the code cleaner. [important]